.github/workflows/migrations.yml

```diff
@@ -5,95 +5,47 @@ on:
-              libxmlsec1-dev \
-              libmaxminddb-dev
-
-        - uses: actions/checkout@v1
-
-        - name: Get changed migration files
-          id: file
-          run: |
-            echo $(git diff --diff-filter=AM --name-only origin/master HEAD)
-            echo "::set-output name=modified::$(git diff --diff-filter=AM --name-only origin/master HEAD | grep 'src/sentry/migrations/')"
-
-        - name: Set up outputs
-          id: config
-          env:
-            MATRIX_INSTANCE: ${{ matrix.instance }}
-          run: |
-            echo "::set-output name=python-version::2.7.17"
-
-        - name: Set up Python ${{ steps.config.outputs.python-version }}
-          uses: actions/setup-python@v2
-          with:
-            python-version: ${{ steps.config.outputs.python-version}}
-
-        - name: Install pip
-          run: |
-            pip install --no-cache-dir --upgrade "pip>=20.0.2"
-
-        - name: Get pip cache dir
-          id: pip-cache
-          run: |
-            echo "::set-output name=dir::$(pip cache dir)"
-
-        - name: pip cache
-          uses: actions/cache@v1
-          with:
-            path: ${{ steps.pip-cache.outputs.dir }}
-            key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
-            restore-keys: |
-              ${{ runner.os }}-pip-
-
-        - name: Install Python Dependencies
-          env:
-            PGPASSWORD: postgres
-          run: |
-            python setup.py install_egg_info
-            pip install wheel # GitHub Actions does not have this installed by default (unlike Travis)
-            pip install -U -e ".[dev]"
-            psql -c 'create database sentry;' -h localhost -U postgres
-            sentry init
-
-        - name: Generate SQL for migration
-          uses: getsentry/action-migrations@v1.0.7
-          env:
-            SENTRY_LOG_LEVEL: ERROR
-            PGPASSWORD: postgres
-          with:
-            githubToken: ${{ secrets.GITHUB_TOKEN }}
-            migration: ${{ steps.file.outputs.modified }}
+  sql:
+    name: Generate SQL
+    runs-on: ubuntu-16.04
+
+    steps:
+      - uses: actions/checkout@v1
+
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 2.7.17
+
+      - name: Setup sentry env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+        with:
+          python: 2
+
+      - name: Get changed migration files
+        id: file
+        run: |
+          echo $(git diff --diff-filter=AM --name-only origin/master HEAD)
+          echo "::set-output name=modified::$(git diff --diff-filter=AM --name-only origin/master HEAD | grep 'src/sentry/migrations/')"
+
+      - name: Get pip cache dir
+        id: pip-cache
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: pip cache
+        uses: actions/cache@v1
```

Comment:
v2 is available for this, you might want to upgrade across the board. Also now that I'm thinking about it, shouldn't this ideally happen as the very last step in a job?

Reply:
Ugh good catch, actually, this is completely wrong and won't do anything because it's supposed to restore the cache 😬

Reply:
Oh.

Reply:
Yeah, I didn't catch that in https://github.com/getsentry/sentry/pull/21203, unfortunate.

Reply:
Just remove this then. Get pip cache dir is also not necessary.